File: app/src/main/java/com/worldventures/dreamtrips/social/ui/reptools/service/analytics/SearchSuccessStoriesAction.java
package com.worldventures.dreamtrips.social.ui.reptools.service.analytics;

import com.worldventures.core.service.analytics.AdobeTracker;
import com.worldventures.core.service.analytics.AnalyticsEvent;
import com.worldventures.core.service.analytics.Attribute;
import com.worldventures.core.service.analytics.BaseAnalyticsAction;

@AnalyticsEvent(action = "rep_tools:success_story", trackers = AdobeTracker.TRACKER_KEY)
public class SearchSuccessStoriesAction extends BaseAnalyticsAction {

   @Attribute("search") final String attribute = "1";
}

File: app/src/main/java/com/worldventures/dreamtrips/social/ui/reptools/service/analytics/SearchSuccessStoriesAction.kt
package com.worldventures.dreamtrips.social.ui.reptools.service.analytics

import com.worldventures.core.service.analytics.AdobeTracker
import com.worldventures.core.service.analytics.Attribute
import com.worldventures.core.service.analytics.BaseAnalyticsAction
import com.worldventures.janet.analytics.AnalyticsEvent

@AnalyticsEvent(action = "rep_tools:success_story", trackers = arrayOf(AdobeTracker.TRACKER_KEY))
class SearchSuccessStoriesAction : BaseAnalyticsAction() {

   @field:Attribute("search") internal val attribute = "1"
}
